dosma/core/io/dicom_io.py

```diff
@@ -687,8 +580,8 @@ def _update_np_dtype(arr: np.ndarray, bit_depth: int):
     dtype_dict = {
         8: [(np.int8, -128, 127), (np.uint8, 0, 255)],
         16: [
-            (np.uint16, 0, 2 ** 16 - 1),
-            (np.int16, -(2 ** 15), 2 ** 15),
+            (np.uint16, 0, 2**16 - 1),
```

Comment:
Formatting is done with black==21.4b2 for now - use `pip install -U black==21.4b2` to update the autoformatter